### ISSUE TITLE
Update when phpactor has been updated

### ISSUE THREAD

**Issue by rtorralba** [phpactor/vscode-phpactor#124]
Check phpactor updates on start or add a update command to get always the las phpactor version

**Comment by zobo** [comment]
Yes, I'm checking where we left of with the release cycle of vscode extension

**Comment by zobo** [comment]
New release out. Seems we need to reconsider automatic updates...
